addons/mail/static/tests/mail_test_helpers_contains.js

```diff
@@ -3,11 +3,11 @@
 import { __debug__, after, afterEach, expect, getFixture } from "@odoo/hoot";
 import { queryAll, queryFirst } from "@odoo/hoot-dom";
 import { Deferred, tick } from "@odoo/hoot-mock";
+import { asyncStep, waitForSteps } from "@web/../tests/web_test_helpers";
 import { isMacOS } from "@web/core/browser/feature_detection";
 import { isVisible } from "@web/core/utils/ui";
-import { step as webStep, assertSteps as webAssertSteps } from "@web/../tests/web_test_helpers";
-export const step = webStep;
-export const assertSteps = webAssertSteps;
+export const step = asyncStep;
+export const assertSteps = waitForSteps;
```

Comment:
Hello, ideally we don't want to re-export this here, if we're good on the new API we might want to update all occurrences in code to use the new one.